exercises/luhn/canonical-data.json

```diff
@@ -170,6 +170,15 @@
         "value": ":9"
       },
       "expected": false
-    }
+    },
+    {
+      "uuid": "401a68ba-98d4-11eb-a8b3-0242ac130003",
```

Comment:
```suggestion
      "uuid": "8b72ad26-c8be-49a2-b99c-bcc3bf631b33",
```

Reply:
I had a one-out-of-two chance and I chose wrong. Thanks! I'll remember that!

Reply:
We used to allow _all_ uuids, but made this restriction so they are really always just completely random.